automata/fa/dfa.py

```diff
@@ -644,7 +695,7 @@ def isfinite(self):
         except exceptions.EmptyLanguageException:
             return True
 
-    def random_word(self, k, *, seed=None):
+    def random_word(self, k: int, *, seed:Optional[int]=None) -> str:
```

Comment:
@eliotwrobson Is the space after the colon normally absent when a default value is involved? I noticed that the `predecessor` function still included the space on line 723 (`strict: bool=True`).

Reply:
Oh the space after the colon is usually present, I was just doing this in a hurry and intended on running an autoformatter before merging.

Reply:
@eliotwrobson Got it, will resolve this thread then, especially since the autoformatter will take care of these things.